base/timing.jl

```diff
@@ -115,41 +124,42 @@ function format_bytes(bytes) # also used by InteractiveUtils
     end
 end
 
-function time_print(elapsedtime, bytes=0, gctime=0, allocs=0, compile_time=0)
+function time_print(elapsedtime, bytes=0, gctime=0, allocs=0, compile_time=0, newline=false)
     timestr = Ryu.writefixed(Float64(elapsedtime/1e9), 6)
-    length(timestr) < 10 && print(" "^(10 - length(timestr)))
-    print(timestr, " seconds")
+    str = length(timestr) < 10 ? (" "^(10 - length(timestr))) : ""
+    str *= string(timestr, " seconds")
```

Comment:
Usually, this is done by doing

```
str = sprint() do io
   print(io, ...)
end
```

to avoid creating so many intermediate strings.

Reply:
Ok, thanks. Updated